content/html/concepts/attributes/terms/class/class.md

```diff
@@ -0,0 +1,82 @@
+---
+Title: 'Classes'
+
+Definition: 'Classes are HTML attributes used to select one or more elements for CSS styling or JavaScript purposes. The class attribute is commonly used for assigning styles to one or more elements. Inside the stylesheet, a class name is selected with the following syntax: css .class-name { }  A class selector is a name preceded by a period (.).'
+
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+
+Tags:
+  - 'Attributes'
+  - 'Classes'
+  - 'Style'
```

Comment:
```suggestion
  - 'Style'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---
```

Reply:
The formatting for the metadata section has several errors. Be sure to always use one of our templates or another entry as a starting point, you should never create an entry from scratch.

Additionally, you should never just copy a paste the written content from another source, even another Codecademy source. All entries should be original, which means you have to phrase the concept and other elements in your own words.